src/utils/post.ts

```diff
@@ -217,3 +217,18 @@ export async function listTags() {
 
 	return tags;
 }
+
+export function getFormattedAuthorsList(authors: string[]) {
+	switch (authors.length) {
+		case 0:
+			return '';
+		case 1:
+			return authors[0];
+		default: {
+			const allButLast = authors.slice(0, -1).join(', ');
+			const lastAuthor = authors[authors.length - 1];
+
+			return `${allButLast} & ${lastAuthor}`;
+		}
+	}
```

Comment:
```suggestion
	return new Intl.ListFormat('en', {
		style: 'long',
		type: 'conjunction',
	}).format(authors);
```

Let's use [`Intl.ListFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) here as it is specifically designed for this use case.